governance/DeprecationMeta.md

```diff
@@ -0,0 +1,27 @@
+# Deprecation Meta
+
+# Overview
+
+This document contains high-level information about O3DE's approach to deprecation.
+
+It details language and terms used when categorizing types of deprecation.
+
+## Philosophy
+
+When modifying existing code that external customers rely on we should strive wherever possible to not break them when making changes.
```

Comment:
Shouldn't we also talk about data, schemas and other things other than code?

Reply:
We definitely should, we just haven't yet, and need to start somewhere. At the moment the focus was on API deprecation (the file ApiDeprecation.md covers this) and full feature/system removals (when a full feature is deprecated, e.g. Slices). We should absolutely have guidance around data/schemas too, but they likely need their own doc and input from other teams. Happy to work with people on this but we have to start somewhere